src/lib.rs

```diff
@@ -1,49 +1,69 @@
+#![deny(unsafe_code)]
+#![warn(clippy::cargo, clippy::missing_docs_in_private_items)]
+#![cfg_attr(doc, warn(rustdoc::all), allow(rustdoc::missing_doc_code_examples))]
+
+//! TODO
+
 use core::cmp::Ordering;
 
 use proc_macro2::{Ident, TokenStream};
-use proc_macro_error::{abort_call_site, proc_macro_error};
-use quote::{format_ident, quote};
+use quote::{format_ident, quote, ToTokens};
 use syn::parse::{Parse, ParseStream};
 use syn::punctuated::Punctuated;
 use syn::spanned::Spanned;
 use syn::{
-    parse_macro_input, Data, DeriveInput, Error, Fields, FieldsNamed, FieldsUnnamed, Token,
-    TraitBound, Type,
+    Data, DeriveInput, Error, Fields, FieldsNamed, FieldsUnnamed, Result, Token, TraitBound, Type,
 };
 
+/// Holds parsed [bounds](TraitBound) and [traits][Traits].
```

Comment:
This looks strange, shouldn't both links have the same syntax?

Reply:
This is a typo.

Reply:
Done.